retis/src/collect/collect.rs

```diff
@@ -108,8 +109,8 @@ pub(crate) struct Collectors {
     tracking_config_map: Option<libbpf_rs::MapHandle>,
     // Retis events factory.
     events_factory: Arc<RetisEventsFactory>,
-    // Did we mount debugfs ourselves?
-    mounted_debugfs: bool,
+    // Did we mount a fs ourselves? If so contains the target dir.
```

Comment:
small nit: this can be tracefs or debugfs only. Isn't `fs` too generic?

Reply:
Fixed!